gui/velociraptor/src/components/forms/form.js

```diff
@@ -72,6 +75,95 @@ export default class VeloForm extends React.Component {
 
         // A Date() object that is parsed from value in local time.
         timestamp: null,
+        multichoices: undefined,
+    }
+
+    componentDidMount = () => {
+        this.source = axios.CancelToken.source();
+        this.fetchArtifacts(this.props.param.artifact_type);
```

Comment:
This should only really be done if this.props.params.type is artifactset - maybe add a check here?